code/parachain/node/src/chain_spec/picasso.rs

```diff
@@ -26,7 +26,7 @@ pub fn genesis_config(
 	treasury: AccountId,
 ) -> picasso_runtime::GenesisConfig {
 	let mut contracts = Vec::new();
-	if let Some(contract) = include_contract(option_env!("CW_XC_GATEWAY_WASM_PATH")) {
+	if let Some(contract) = option_env!("CW_XC_GATEWAY_WASM_PATH") {
 		contracts.push(contract);
 	}
 	if let Some(contract) = include_contract(option_env!("CW_XC_INTERPRETER_WASM_PATH")) {
```

Comment:
Sorry, missed this one.  This should also drop `include_contract` call and than you can drop the function altogether.

```suggestion
	if let Some(contract) = option_env!("CW_XC_INTERPRETER_WASM_PATH") {
```